main/config.py

```diff
@@ -0,0 +1,19 @@
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
```

Comment:
I don't see how we can setup multi-environment configuration

Reply:
I haven't figured how to achieve `multi-environment configuration`. I need help on this.